specs/pvs/corla.pvs

```diff
@@ -1001,23 +1050,29 @@ END rlas
 
 %+BEGIN cvrs
 
-%%% \section{Cast Vote Records}
+%%% \subsection{Cast Vote Records}
 
 %%% Cast vote records, also known as CVRs, are the digital
 %%% interpretations of paper ballot records by a computer. They
 %%% frequently, but not always, contain an interpretation of all voter
-%%% choicdes in all contests on a ballot. CVRs are sometimes
+%%% choices in all contests on a ballot. CVRs are sometimes
 %%% syntactically written as comma-separated values, and other times
 %%% in plain English. Some CVRs use a election-specific encoding
 %%% scheme to represent choices (e.g., a '1' means "John Doe"); others
 %%% use plain English.
 %%%
+%%% CVRs may (and in the case of CVRs exported from Colorado's Dominion system, do)
+%%% contain information about the ballot beyond the voter choices. For 
+%%% example, ballot style, precinct id and ballot type (in-person vs. mail)
+%%% may be included.
+%%%
```

Comment:
`CountingGroup` is the one that are "Mail" or "In Person". `BallotType` is the ballot style, which includes a precinct number which is a big problem from a CORA standpoint. We would prefer that all counties upload a CVR which has all the precinct information left out: `PrecinctPortion`, the precinct part of `BallotType`, and `ImprintedId`.